package.json

```diff
@@ -6,10 +6,27 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1"
   },
```

Comment:
## Add scripts here to smoothly run and test the application
- [ ] You are required to include the required scripts to this `package.json`  file. We need to aim for at least 2 scripts, one for development and another one for production and both needs to specify in each mode Webpack should run: As in the example below 👇 

```javascript
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
  },
``` 

## Optional Change
- [ ] Also while at it, you are welcome to add the `webpack-dev-server` package to your dependencies and add an additional script. So in overall, we might have something that looks like this 👇 in the `package.json` file:

```javascript
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },
```